src/header/hooks/useMenuItems.ts

```diff
@@ -0,0 +1,62 @@
+import { useContext } from 'react';
+import { useLocation } from 'react-router-dom';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import { AppContext } from '@edx/frontend-platform/react';
+import { getConfig } from '@edx/frontend-platform';
+
+import { ROUTES } from '../../routes';
+import { programsUrl } from '../../utils';
+import { AppContextTypes, MenuItem } from '../types';
+import messages from '../messages';
```

Comment:
[nit] Let's add an empty line before
 
```suggestion

import messages from '../messages';
```

Reply:
✅